platform/opensuse/kanidm-unixd-tasks.service

```diff
@@ -3,8 +3,10 @@
 
 [Unit]
 Description=Kanidm Local Tasks
-After=chronyd.service ntpd.service network-online.target suspend.target kanidm-unixd.service
-Wants=kanidm-unixd.service
```

Comment:
I'd say leave the `Wants` in here, just to make sure it *tries* to start `kanidm-unixd.service` if it's not running.

> Units listed in this option will be started if the configuring unit is. However, if the listed units fail to start or cannot be added to the transaction, this has no impact on the validity of the transaction as a whole, and this unit will still be started. This is the recommended way to hook the start-up of one unit to the start-up of another unit.

Reply:
Wants does try to start it iirc, but it just means that if unixd fails, tasks itself doesn't fail so it still keeps going.

Reply:
Reading the docs `Upholds` is a superset of what `Wants` entails, i.e. it does the exact same startup chaining in addition to monitoring:
> Configures dependencies similar to Wants=, but as long as this unit is up, all units listed in Upholds= are started whenever found to be inactive or failed, and no job is queued for them.

I've improved the documenting comment a bit since someone would definitely wonder later why there isn't a `Wants`.